ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -40,16 +40,12 @@ let stunnel_m = Mutex.create ()
 
 let restart_stunnel_nomutex ~__context ~accept =
   info "Restarting stunnel (accepting connections on %s)" accept;
-  let back_compat ~__context =
-    if Stunnel.is_legacy_protocol_and_ciphersuites_allowed ()
-    then [ "back_compat_6_5" ]
-    else []
+  let xapissl_args = [ "restart"
+                     ; accept
+                     ; "permfile=" ^ !Xapi_globs.server_cert_path
```

Comment:
is `permfile` the correct name for the parameter? It looks like it should be `pemfile` since it's the path of the pem (certificate file)

Reply:
i'll fix this when i convert this script to ocaml